q2_emperor/_plot.py

```diff
@@ -16,17 +16,42 @@
 
 TEMPLATES = pkg_resources.resource_filename('q2_emperor', 'assets')
 
-
-def plot(output_dir: str, pcoa: skbio.OrdinationResults,
-         metadata: qiime2.Metadata, custom_axes: str=None) -> None:
+def _generic_plot(output_dir: str, master: skbio.OrdinationResults,
```

Comment:
By default q2templates plots the calling function that renders the viz, resulting in this:

The following patch squares this away (I can push this as a commit to this feature branch if that is okay):

```patch
index 50660f5..9e488fe 100644
--- a/q2_emperor/_plot.py
+++ b/q2_emperor/_plot.py
@@ -16,9 +16,11 @@ from emperor import Emperor

 TEMPLATES = pkg_resources.resource_filename('q2_emperor', 'assets')

+
 def _generic_plot(output_dir: str, master: skbio.OrdinationResults,
                   metadata: qiime2.Metadata,
-                  other_pcoa: skbio.OrdinationResults, custom_axes: str=None):
+                  other_pcoa: skbio.OrdinationResults, plot_name,
+                  custom_axes: str=None):

     mf = metadata.to_dataframe()

@@ -41,17 +43,18 @@ def _generic_plot(output_dir: str, master: skbio.OrdinationResults,
         fh.write(html)

     index = os.path.join(TEMPLATES, 'index.html')
-    q2templates.render(index, output_dir)
+    q2templates.render(index, output_dir, context={'plot_name': plot_name})


 def plot(output_dir: str, pcoa: skbio.OrdinationResults,
          metadata: qiime2.Metadata, custom_axes: str=None) -> None:
     _generic_plot(output_dir, master=pcoa, metadata=metadata, other_pcoa=None,
-                  custom_axes=custom_axes)
+                  custom_axes=custom_axes, plot_name='plot')


 def procrustes_plot(output_dir: str, reference_pcoa: skbio.OrdinationResults,
                     other_pcoa: skbio.OrdinationResults,
                     metadata: qiime2.Metadata, custom_axes: str=None) -> None:
     _generic_plot(output_dir, master=reference_pcoa, metadata=metadata,
-                  other_pcoa=other_pcoa, custom_axes=custom_axes)
+                  other_pcoa=other_pcoa, custom_axes=custom_axes,
+                  plot_name='procrustes_plot')
diff --git a/q2_emperor/assets/index.html b/q2_emperor/assets/index.html
index da3626d..1fefd9c 100644
--- a/q2_emperor/assets/index.html
+++ b/q2_emperor/assets/index.html
@@ -1,5 +1,7 @@
 {% extends 'base.html' %}

+{% block title %}q2-emperor : {{ plot_name }}{% endblock %}
+
 {% block head %}

 <style media="screen">
```

Reply:
Uh, looks like I forgot to attach my critical screenshot here. The index.html title tag shows the private helper name instead of the Action name